content/_data/events/2022-09-27-florida.adoc

```diff
@@ -0,0 +1,6 @@
+---
+name: "Jenkins Contributor Summit @ DevOps World"
```

Comment:
```suggestion
title: "Jenkins Contributor Summit @ DevOps World"
```

Reply:
Thanks timja for the suggested change. I'm wondering if it would be worthwhile for us to update the guide  at https://github.com/jenkins-infra/jenkins.io/blob/master/CONTRIBUTING.adoc#adding-an-event  - chances are I might not remember for next time and I may make the same mistake :P